ocs_ci/ocs/resources/mcg.py

```diff
@@ -593,7 +593,60 @@ def create_namespace_resource(
             },
         )
         logger.info(f"result from RPC call: {result}")
+        return target_bucket_name
+
+    def create_namespace_store(
+        self, namespace_store_name, region, cld_mgr, cloud_uls_factory, platform
+    ):
+        """
+        Creates a new namespace store
+
+        Args:
+            namespace_store_name (str): The name to be given to the new namespace store
+            region (str): The region name to be used
+            cld_mgr: A cloud manager instance
+            cloud_uls_factory: The cloud uls factory
+            platform (str): The platform resource name
+
+        Returns:
+            str: The name of the created target_bucket_name (cloud uls)
+        """
+        # Create the actual target bucket on AWS
+        uls_dict = cloud_uls_factory({platform: [(1, region)]})
+        target_bucket_name = list(uls_dict[platform])[0]
+
+        ns_data = templating.load_yaml(constants.MCG_NAMESPACESTORE_YAML)
```

Comment:
I think it'd be best to define `NS` as `NAMESPACE` and `NSS` as `NAMESPACESTORE`.
Can you please refactor accordingly?

Reply:
sure, fixed